src/MongoDB/ReadPreference.stub.php

```diff
@@ -12,30 +12,35 @@ final class ReadPreference implements \MongoDB\BSON\Serializable, \Serializable
     /**
      * @var int
      * @cvalue MONGOC_READ_PRIMARY
+     * @deprecated
```

Comment:
This will need to be sync with 3rd party stubs.

Reply:
Yes, I'll take care of those alongside the documentation